configure.host

```diff
@@ -140,6 +140,11 @@ case "${host}" in
 	SOURCES="ffi.c sysv.S"
 	;;
 
+  loongarch64-*-*)
+	TARGET=LOONGARCH; TARGETDIR=loongarch
```

Comment:
Judging from the implementation, I'm not entirely sure about your intention. Do you want to support 32-bit and 64-bit ABIs in the outset? Or do you want to postpone 32-bit support because the 32-bit cases seem unpolished to some degree? If you decide to only support 64-bit in this PR then this should probably be `TARGET=LOONGARCH64; TARGETDIR=loongarch64` instead, else the matching arm should say `loongarch*-*-*)`. Either way this needs change.